src/main/java/duckie/Duckie.java

```diff
@@ -0,0 +1,70 @@
+package duckie;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.Scanner;
+
+import duckie.command.Command;
+import duckie.exception.DuckieException;
+import duckie.task.TaskList;
+
+/**
+ * Main file for the chatbot Duckie
```

Comment:
I think is good to have the header <h></h> and @author tag in the JavaDoc of the class.